src/components/rpc_base/include/rpc_base/rpc_base_json_inl.h

```diff
@@ -287,6 +287,61 @@ Json::Value Array<T, minsize, maxsize>::ToJsonValue() const {
   return array;
 }
 
+////////// Array to be instantiated with Enum<T>
+
+// Non-const version
+template <typename E, size_t minsize, size_t maxsize>
+Array<Enum<E>, minsize, maxsize>::Array(Json::Value* value)
+    : CompositeType(InitHelper(value, &Json::Value::isArray)) {
+  if (value) {
+    if (value->isArray()) {
```

Comment:
@LuxoftAKutsan I believe these two conditions could be merged to one using &&

Reply:
@AKalinich-Luxoft Fixed in d1308970ca2afad7de251f6ef744d94c4422cb42